apps/nestjs-wallet/src/vc-api/exchanges/entities/transaction.entity.ts

```diff
@@ -27,6 +34,9 @@ export class TransactionEntity {
         reviewStatus: PresentationReviewStatus.pending
       };
     }
+    this.callback = callback?.map((c) => {
+      return { status: 'not called', ...c };
```

Comment:
```suggestion
      return { status: 'not called', url: c.url };
```
I would do it this way. In case that object `c` would contain something more than just an url.

Reply:
Thanks, I aligned the types so that I can assign directly.